integration-tests/ignore/src/test/java/io/quarkiverse/openapi/generator/it/IgnoreTest.java

```diff
@@ -1,12 +1,12 @@
 package io.quarkiverse.openapi.generator.it;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
I know this in unrelated, but every time I run maven install it got changed, so I guess it should be committed to avoid the same happening to other contributors.